README.md

```diff
@@ -0,0 +1,162 @@
+[![Build Status](https://travis-ci.org/dask/dask.svg?branch=master)](https://travis-ci.org/dask/dask)
+[![Coverage Status](https://coveralls.io/repos/dask/dask/badge.svg)](https://coveralls.io/r/dask/dask)
+[![Documentation Status](http://readthedocs.org/projects/dask/badge/?version=latest)](http://dask.pydata.org/en/latest/)
+[![Join the chat at https://gitter.im/dask/dask](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/dask/dask?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Version Status](https://img.shields.io/pypi/v/dask.svg)](https://pypi.python.org/pypi/dask/)
+
+# Dask
+
+*Dask is a flexible parallel computing library for analytic computing.*  
+
+See [documentation](http://dask.pydata.org/en/latest/) for more information.
+
+
+### Sneak Peek
+
+A simple computation
+
+```python
+
+def inc(i):
+   return i + 1
+
+def add(a, b):
+   return a + b
+
+x = 1
+y = inc(x)
+z = add(y, 10)
+```   
+
+is representable like:
+
+
+```python
+dsk = {'x': 1,
+      'y': (inc, 'x'),
+      'z': (add, 'y', 10)}
+```        
+
+which happens to be a graph:
+
+![alt text](https://github.com/dask/dask/blob/master/docs/source/_static/dask-simple.png "A simple dask dictionary")
+
+where the computation of `z` is parallelizable
+
+
+```python
+
+from dask import get
+
+z = get(dsk, 'z')
+```
+
+
+# Collections and Schedulers
+
+To handle more complex cases there are a variaty of abstractions and schedulers available.
```

Comment:
Sure, it is just a skeleton. A section like "When is dask a good choice" or "Computations where dask shines" containing a short list: bigger-than-memory, distributed etc. should go to the top of the README. Can You edit this PR?